docs/notes-pp.md

```diff
@@ -5,4 +5,52 @@ of the permutation polynomial evaluations.
 Permutation polynomials
 ======================
 
-A permutation
\ No newline at end of file
+A permutation polynomial is bijective 
+function which describes a permutation, 
+of a group of elements. 
+
+The purpose of constructing these 
+polynomials is often for analysis,
+which is done on a permutated set. 
+A permutated set is a different 
+linear arrangement of elements. 
+
+The linearity of these is how 
+permutations differ from other 
+means of selection from a set. 
+
+For example: 
+
+\begin{aligned}
+{1,2,3}
+\end{aligned}
+
+is a number set which can have 
+only six permutations, namely:
+
+\begin{aligned}
+{1,2,3}
+{3,2,1}
+{2,1,3}
+{2,3,1}
+{1,3,2}
+{3,1,2}
+\end{aligned}
+
+This example satisfies the two 
+main properties which define set
+permutations:
+
+1. The number of permutations is at most 
+n! for n elements. 
+
+2. The permutation is a bijection onto
+itself. 
+
+These basic properties are used in 
+the construction of zero knowledge 
+proofs and when the individual 
+permutations are described, they 
+are often compounded into 
+polynomials which describe these many
+functions as polynomials over sets.
```

Comment:
To me, this reads as: Permutations are compounded into polynomials which describe many functions as polynomials over sets?